styleguide/components/41-organisms/timeline/_timeline.scss

```diff
@@ -0,0 +1,397 @@
+        top: 0;
+        left: 0;
+        height: 1.3rem;
+        border-left: 3px solid;
+        content: '';
+
+        @include tablet {
+          height: 2.3rem;
+        }
+      }
+
+      // Hide video dots on last timeline-slot's last definition description.
+      .responsive-video::before,
+      .audio-description::after {
+        display: none;
+      }
+    }
+
+    // Timeline-slot title styling.
+    &-title {
+      &[aria-expanded="true"] {
+        @include icon('minus');
+      }
+
+      &[aria-expanded="false"] {
+        @include icon('plus');
+      }
+
+      position: relative;
+      padding: 0;
+      border: 0;
+      background: transparent;
+
+      h3 {
+        width: 100%;
+        margin-bottom: .2rem;
+        font-size: .9rem;
+        text-align: left;
+      }
+
+      &::before {
+        position: absolute;
+        left: -1.6rem;
+        font-size: 1.6em;
+      }
+    }
+
+    &-date {
+      display: block;
+      width: 100%;
+    }
+
+    &-content {
+      width: 100%;
+    }
+
+    > dt {
+      @extend %h5;
+
+      position: relative;
+      width: 100%;
+      margin: 0;
+      padding: 1.5rem 5% .5rem;
+
+      @include tablet {
+        position: absolute;
+        top: 1.9rem;
+        left: 0;
+        width: 11.5rem;
+        padding: 0 1.2rem 0 0;
+        text-align: right;
+      }
+
+      &::before {
+        @include theme('border-color', 'color-primary', 'timeline-line-color');
+
+        position: absolute;
+        top: 0;
+        left: 1.5rem;
+        height: 1rem;
+        border-left: 3px solid;
+        content: '';
+        @include tablet {
+          display: none;
+        }
+      }
+    }
+
+    > dd {
+      @include theme('border-color', 'color-primary', 'timeline-line-color');
+
+      display: flex;
+      position: relative;
+      flex-wrap: wrap;
+      width: 100%;
+      margin: 0 0 0 1.5rem;
+      padding: .5rem calc(#{$gutter-width/2}) 2rem 2.5rem;
+      border-left: 3px solid;
+
+      @include tablet {
+        margin: 0 0 0 11.5rem;
+        padding: 2rem calc(#{$gutter-width/2}) 2rem 5rem;
+      }
+
+      &::before {
+        @include theme('background-color', 'color-secondary', 'timeline-slot-indicator-background');
+        @include theme('border-color', 'color-primary', 'timeline-slot-indicator-border-color');
+
+        display: block;
+        position: absolute;
+        top: .9rem;
+        left: -.5rem;
+        width: .9rem;
+        height: .9rem;
+        border: 3px solid;
+        border-radius: 100%;
+        content: '';
+        z-index: 2;
+
+        @include tablet {
+          top: 2.3rem;
+        }
+      }
+
+      &.active::before {
+        @include theme('background-color', 'color-primary', 'timeline-slot-active-indicator-background');
+      }
+
+      // Video component styling inside a timeline-slot.
+      .video-wrapper {
+        position: relative;
+        width: calc(100% + 4.6rem + 4px);
```

Comment:
What is the origin of this 4px? It's unclear.

Reply:
True, It's from the timeline border. I'll change this.